src/modules/components/dao/daoDataListItem/daoDataListItem.tsx

```diff
@@ -0,0 +1,45 @@
+import type React from 'react';
+import { DataList, Heading, Icon, IconType } from '../../../../core';
+import { DaoAvatar, type IDaoAvatarProps } from '../daoAvatar';
+
+export interface IDaoDataListItemProps extends IDaoAvatarProps {
+    description?: string;
+    daoAddressOrEns?: string;
+    plugin?: string;
+    network?: string;
+}
+
+export const DaoDataListItem: React.FC<IDaoDataListItemProps> = (props) => {
+    const { name, src, description, network, plugin = 'token-based', daoAddressOrEns } = props;
+
+    return (
+        <DataList.Item role="listitem">
```

Comment:
Update this to pass all the other props passed to the component. Moreover, the component will have a `link` role when the `href` property is set, we don't need to add another role
```suggestion
        <DataList.Item {...otherProps}>
```